t/photostorage/s3.t

```diff
@@ -112,7 +113,9 @@ FixMyStreet::override_config {
             $create_bucket_called = 1;
         });
 
-        ok !$s3->init(), "PhotoStorage::S3::init failed";
+        warning_like {
+            $s3->init();
```

Comment:
Does this no longer test the return value of `init()`, or does `warning_like` handle that?

Reply:
Hmm, you're probably right that it no longer does, not sure if I had an issue combining with `warning_like` or just missed it quite possibly.